resources/tap/main.brs

```diff
@@ -0,0 +1,7 @@
+' Initializes a tap instance and prints out the plan.
```

Comment:
This fixes a previous bug I just noticed. When I converted `roca_main` to TS, I put `tap.brs` in executable script format, instead of pulling it into the execution scope, because I thought that it only self-referenced.

But, I missed the `roca.log()` function, that also uses `tap.brs`. So now, the executable part is still run (this file), but the `Tap` object and function definitions are pulled into scope.

Reply:
The reason I'm fixing here is because `roca.log()` _uses_ the `printExtras` function that this PR modifies.